src/napari_imagej/_tests/test_ntypes.py

```diff
@@ -1,9 +1,16 @@
 from typing import Any, Dict, List
+from jpype import JArray, JDouble
 import pytest
 import numpy as np
+from scyjava import jimport
+import imagej
 from napari_imagej._ntypes import _labeling_to_layer, _layer_to_labeling
 from labeling.Labeling import Labeling
-from napari.layers import Labels
+from napari.layers import Labels, Shapes
+
+@pytest.fixture()
```

Comment:
We have one of these in other testing files, yeah? Can we just use that?

Reply:
We don't have it in other files within this project @gselzer, but we can still put it in `conftest.py`.

I did this in 0e5398f.